boards/wiznet/w5500_evb_pico2/doc/index.rst

```diff
@@ -0,0 +1,269 @@
+.. _w5500_evb_pico2:
+
+Wiznet W5500 Evaluation Pico
+############################
+
+Overview
+********
+
+W5500-EVB-Pico is a microcontroller evaluation board based on the Raspberry
```

Comment:
Pico --> Pico2? Looks like you need to do a global pass just like with RP2040 vs RP2350

Reply:
Ah, this is still draft...
I haven't written the documentation yet.
I plan to update it based on rp2040.

Reply:
Sorry didn't notice it was in draft, my bad!

Reply:
Don't mind. Thank you for caring.

Reply:
I have made some general revisions and removed the draft. Please review it again.